File: hw/astudillo/U3/Hw19Composite/CompositePython/composite_pattern/model/manager.py
from .supervisor import Supervisor

class Manager(Supervisor):
    def __init__(self, name=None):
        super().__init__(name)
        self.title = "Manager"
    
    def state_name(self, title=None, name=None):
        if title is None and name is None:
            super().state_name(self.title, self.name)
        else:
            super().state_name(title, name)

File: hw/fuentes/U3/HW20COMPOSITE/Composite/src/main/java/ec/edu/espe/enterprise/model/Manager.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.enterprise.model;

/**
 *
 * @author jorge
 */
public class Manager extends Supervisor{
    
    public Manager(String aName){
        this();
        name=aName;
    }
    
    public Manager(){
        super();
        title="Manager";
        
    }
    
    @Override
    public void stateName(){
        super.stateName();
    }
    
}
